src/app/client/src/app/modules/dashboard/components/data-chart/data-chart.component.html

```diff
@@ -22,62 +33,45 @@
           </sui-select-option>
         </sui-select>
       </div>
-      <div class="field">
-        <button appTelemetryInteract [telemetryInteractObject]="telemetryInteractObject"
-          [telemetryInteractEdata]="setTelemetryInteractEdata('reset-filters')"
-          [telemetryInteractCdata]="telemetryCdata" class="sb-btn sb-btn-outline-primary sb-btn-normal"
-          *ngIf="filters.length" (click)="resetFilter()">
-          {{resourceService?.frmelmnts?.btn?.reset}} {{resourceService?.frmelmnts?.lbl?.filters  }}
-        </button>
-      </div>
+   
+    <button *ngIf="!showFilters && isUserReportAdmin && chartConfig.id" [ngClass]="{'hide': hideElements}" appTelemetryInteract [telemetryInteractObject]="telemetryInteractObject"
```

Comment:
This button now needs to be shown based on two criteria's only  -
1. if user has REPORT_ADMIN role
2.  chart has a chart id

Please remove the !showFilters condition

Reply:
it was there in the previous code version because we'd 2 placement of the button based on whether the chart has filters or not ...

Reply:
resolved